src/Template/Error/missing_action.ctp

```diff
@@ -26,36 +26,51 @@ if (!empty($prefix)) {
     $prefixNs = '\\' . $prefix;
     $prefix .= DIRECTORY_SEPARATOR;
 }
+
+// Cake\Controller\Exception\MissingActionException support
+if (isset($controller)) {
+    $baseClass = $namespace . '\Controller\AppController';
+    $extends = 'AppController';
+    $type = 'Controller';
+    $class = $controller;
+}
+// Cake\Mailer\Exception\MissingActionException support
```

Comment:
I had to read it twice.

I would word it differently:
``Controller related MissingActionException support``
and
``Mailer related MissingActionException support``

This way, the relevant information doesn't get lost in the long namespace string.